recipes/mender-client/debian-master/control

```diff
@@ -3,8 +3,6 @@ Maintainer: The Mender Team <mender@northern.tech>
 Section: admin
 Priority: optional
 Standards-Version: 4.0.0
-Depends: mender-update, mender-auth
-# debian buster Depends: mender-update, mender-auth
 Build-Depends-Indep: cmake, debhelper (>= 10)
 Build-Depends: liblmdb-dev (>= 0.9.7), libc6-dev (>= 2.14), libboost-dev (>= 1.67.0), libboost-filesystem-dev (>= 1.67.0), libboost-log-dev (>= 1.67.0), libboost-regex-dev (>= 1.67.0), libboost-chrono-dev (>= 1.67.0), libboost-thread-dev (>= 1.67.0), libboost-atomic-dev (>= 1.67.0),  libssl-dev (>= 1.1.1), libarchive-dev (>= 3.2.2), libdbus-1-dev
```

Comment:
I guess now, technically, the `boost` versions here are not correct either.

Reply:
You are right. But changing it will required yet another special handling for old Debian and old Ubuntu to ignore these. I think we let them be.